test/wrappers/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+ #
+ # CMakeLists.txt
+ # Copyright (c) 2019 Hewlett Packard Enterprise Development, LP. All rights reserved.
+ # Redistribution and use in source and binary forms, with or without modification, are permitted provided
+ # that the following conditions are met:
+ # 1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+ # 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer
+ #    in the documentation and/or other materials provided with the distribution.
+ # 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products
+ #    derived from this software without specific prior written permission.
+ #
+ #    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING,
+ #    BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ #    SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ #    DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ #    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+ #    OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ #
+ # See https://spdx.org/licenses/BSD-3-Clause
+ #
+ #
+
+enable_testing()
+include_directories(${PROJECT_BINARY_DIR}/test) # private headers (test)
+add_custom_target(wrappers ${CMAKE_CTEST_COMMAND} --force-new-ctest-process)
```

Comment:
Isnt it better to name it c-wrappers-test in place of make wrappers

Reply:
The 'wrapper' directory is added to host c-wrappers as well as any other wrappers we might introduce in the future. Hence, the make is kept 'wrappers' in place of 'c-wrappers-test'.